src/testers.jl

```diff
@@ -59,6 +77,35 @@ function _make_fdm_call(fdm, f, ȳ, xs, ignores)
     return (args...,)
 end
 
+"""
+    _make_jvp_call(fdm, f, xs, ẋs, ignores)
+
+Call `FiniteDifferences.jvp`, with the option to ignore certain `xs`.
+
+# Arguments
+- `fdm::FiniteDifferenceMethod`: How to numerically differentiate `f`.
+- `f`: The function to differentiate.
+- `xs`: Inputs to `f`, such that `y = f(xs...)`.
+- `ẋs`: Derivative of `xs` w.r.t. some real number `t`.
+- `ignores`: Collection of `Bool`s, the same length as `xs` and `ẋs`.
+   If `ignores[i] === true`, then `ẋs[i]` is ignored for derivative estimation.
+
+# Returns
+- `∂Ω`: Derivative of output w.r.t. `t` estimated by finite differencing.
+"""
+function _make_jvp_call(fdm, f, xs, ẋs, ignores)
+    f2 = _wrap_function(f, xs, ignores)
+
+    ignores = collect(ignores)
+    args = Any[nothing for _ in 1:length(xs)]
+    all(ignores) && return (args...,)
```

Comment:
we never use args again do we?
Shall we just directly construct the tuple, and save on some allocations?
```suggestion
    all(ignores) && return ntuple(_->nothing, length(xs))
```